image/discourse_dev/Dockerfile

```
@@ -12,6 +12,9 @@ ADD sudoers.discourse /etc/sudoers.d/discourse
 
 RUN sudo -u discourse bundle config set --global path /home/discourse/.bundle/gems
 
+# Add user-install ruby gems to PATH
+RUN echo 'PATH="$(ruby -r rubygems -e "puts Gem.user_dir")/bin:$PATH"' >>/home/discourse/.profile
```

we would usually add whitespace there, right?

```suggestion
RUN echo 'PATH="$(ruby -r rubygems -e "puts Gem.user_dir")/bin:$PATH"' >> /home/discourse/.profile
```